lib/config/types.ts

```diff
@@ -132,6 +132,10 @@ export interface RepoGlobalConfig {
   dockerUser?: string;
   dryRun?: DryRunConfig;
   executionTimeout?: number;
+  gerritLabelMapping?: {
+    stabilityDaysLabel?: string;
```

Comment:
`stabilityDays` is now `minimumReleaseAge`. Please update the docs and code. 😉 

https://docs.renovatebot.com/configuration-options/#minimumreleaseage